Library/Homebrew/cask/audit.rb

```diff
@@ -317,5 +319,19 @@ def check_download
     rescue => e
       add_error "download not possible: #{e.message}"
     end
+
+    def check_https_availability
+      return unless download
+      unless cask.url.to_s.empty? || cask.url.using
```

Comment:
Can this be made into an `if`; it's easier to parse with multiple conditions. Also, all uses of `to_s.empty?` can now use `.present?` or `.blank?`

Reply:
```
Offenses:

Library/Homebrew/cask/audit.rb:324:7: C: [Corrected] Favor unless over if for negative conditions.
      return if !download
      ^^^^^^^^^^^^^^^^^^^
Library/Homebrew/cask/audit.rb:325:119: C: Line is too long. [123/118]
      check_url_for_https_availability(cask.url, user_agents: [cask.url.user_agent]) if !cask.url.blank? && !cask.url.using
                                                                                                                      ^^^^^

723 files inspected, 2 offenses detected, 1 offense corrected
```